File: Angular_DotNetCore/Models/Airline.cs
/// <summary>
/// Airline class.
/// </summary>
public class Airline
{
    /// <summary>
    /// Name of the airline.
    /// </summary>
    public string Name { get; set; }
    /// <summary>
    /// Alias of the airline.For example, All Nippon Airways is commonly known as "ANA".
    /// </summary>
    public string Alias { get; set; }
    /// <summary>
    /// True if the airline is or has until recently been operational, otherwise it is defunct.
    /// </summary>
    public bool Active { get; set; }
}

File: Angular_DotNetCore/ClientApp/app/models/airline.ts
/**Airline class. */
export class Airline {
    /**
     * Constructor.
     * @param name Name of the airline.
     * @param alias Alias of the airline.For example, All Nippon Airways is commonly known as "ANA".
     * @param active True if the airline is or has until recently been operational, otherwise it is defunct.
     */
    constructor(
        name: string = null,
        alias: string = null,
        active: boolean = null,
    ) {
        this.name = name;
        this.alias = alias;
        this.active = active;
    }
    /**Name of the airline. */
    public name: string;
    /**Alias of the airline.For example, All Nippon Airways is commonly known as "ANA". */
    public alias: string;
    /**True if the airline is or has until recently been operational, otherwise it is defunct. */
    public active: boolean;
}